app/app.rb

```diff
@@ -21,6 +21,9 @@
 require_relative "tint_omniauth" # Monkeypatch
 require_relative "helpers"
 
+ENV["GIT_COMMITTER_NAME"] = "Tint"
```

Comment:
Does this git gem automatically use these environment variables? I don't see them explicitly used anywhere.